docs/config.md

```diff
@@ -2,20 +2,18 @@
 
 To use workflow with WSO2 Identity Server 7 onwards, you need to follow below steps. 
 
-## Instructions to enable workkflow
+# Instructions to enable workkflow
 
 1. Unzip the zip file.
-2. Run identity workflow-related db scripts against the identity database known as IDENTITY_DB (WSO2IDENTITY_DB.mv.db).
-3. Copy bps-datasources.xml to <IS_HOME>/repository/conf/datasources.
-4. Run bps-related scripts against the database called JPA_DB (jpadb.mv.db).
-   More on:
-   https://is.docs.wso2.com/en/latest/deploy/work-with-databases/, 
-   https://is.docs.wso2.com/en/latest/deploy/change-datasource-bpsds/
-
+2. Run identity workflow-related db scripts against the identity  <IS_HOME>/repository/database/ IDENTITY_DB(WSO2IDENTITY_DB.mv.db).
+   If the production db run the scripts against it after configuring it from [here](https://is.docs.wso2.com/en/next/deploy/configure/databases/carbon-database/).
+3. Copy bps-datasources.xml to  <IS_HOME>/repository/conf/datasources.
+4. Copy the provided file (jpadb.mv.db) or Run bps-related scripts against the database in <IS_HOME>/repository/database/JPA_DB(jpadb.mv.db). If you need to change to a different db follow this [guide](https://is.docs.wso2.com/en/next/deploy/configure/databases/carbon-database/change-datasource-bpsds/) and copy bps-datasources.xml.j2 to <IS_HOME>/repository/resources/conf/templates/repository/conf/datasources/bps-datasources.xml.j2
 5. Copy 4 XML files in xml folder to <IS_HOME>/repository/conf.
-6. Add jars in the dropins folder to <IS_HOME>/repository/components/dropins.
+6. Add all jars in the dropins folder to <IS_HOME>/repository/components/dropins.
 7. Copy weblib jars to <IS_HOME>/repository/deployment/server/webapps/api/WEB-INF/lib.
-8. Add listener and handler to <IS_HOME>/repository/conf/deployment.toml
+8. To enable APIs related to approval  add <import resource="classpath:META-INF/cxf/user-approval-v1-cxf.xml"/> under beans starting tag and add <bean class="org.wso2.carbon.identity.rest.api.user.approval.v1.MeApi"/> under <jaxrs:server id="users" address="/users/v1"> to beans.xml located in <IS_HOME>/repository/deployment/server/webapps/api/WEB-INF/beans.xml.
```

Comment:
```suggestion
8. To enable APIs related to approval  add `<import resource="classpath:META-INF/cxf/user-approval-v1-cxf.xml"/>`  under beans starting tag and add <bean class="org.wso2.carbon.identity.rest.api.user.approval.v1.MeApi"/> under <jaxrs:server id="users" address="/users/v1"> to beans.xml located in <IS_HOME>/repository/deployment/server/webapps/api/WEB-INF/beans.xml.
```
```suggestion
8. To enable APIs related to approval  add <import resource="classpath:META-INF/cxf/user-approval-v1-cxf.xml"/> under beans starting tag and add <bean class="org.wso2.carbon.identity.rest.api.user.approval.v1.MeApi"/> under <jaxrs:server id="users" address="/users/v1"> to beans.xml located in <IS_HOME>/repository/deployment/server/webapps/api/WEB-INF/beans.xml.
```

Reply:
mentioned class is not visible in the readme file. better to check this as well: https://github.com/wso2-extensions/identity-workflow-impl-bps/blob/f933fb77a1f60bb70957d642ebae28ed443df07d/docs/config.md